hera_mc/tests/test_sys_handling.py

```diff
@@ -63,7 +63,7 @@ def test_correlator_info(self):
 
         self.assertEqual(stn_types[index], 'herahexw')
 
-        self.assertEqual(corr_inputs[index], ('DF8B2', 'DF8B1'))
+        self.assertEqual(corr_inputs[index], ('input>DF8B2', 'input>DF8B1'))
```

Comment:
This confuses me. Why do we want `'input>'` in the string?

Reply:
For the new correlator we need the port and the part number, for the old correlator we only needed the part number and the old "port" was 'input'

Reply:
ok. I find the terminology a little confusing, but I can probably get used to it.